### PR TITLE
feat(kinder): add announcement bar

### PR DESCRIPTION
**Pull request by eihabkhan**
## JIRA Ticket

Ticket: [TH_X](https://youcanshop.atlassian.net/browse/TH_X).

## Prerequisites

* [ ] Check this branch locally and run `pnpm run release`
* [ ] 3 new files will be generated in this format `theme name + date + .zip`
* [ ] Upload generated file locally or in seller-area test env

## QA Steps

* [ ] Go to storefront
* [ ] Ensure you see the announcement bar
* [ ] Open a new tab with theme editor
* [ ] Make sure each setting applies properly
* [ ] Test edge cases: No annon, 1 annon, max annon, etc

## Note

Leave empty when you have nothing to say.

### CODE REVIEW

File: themes/kinder/locales/ar.schema.json [hunk 1]
```diff
@@ -83,6 +84,8 @@
     },
     "colors": {
       "main_color": "اللون الرئيسي",
+      "bg_color": "لون الخلفية",
+      "fg_color": "لون النص",
```
Comment (ibrilBadreddine) [inline themes/kinder/locales/ar.schema.json:88]: j'aime, je partage

File: themes/kinder/sections/announcement-bar.liquid [hunk 2]
```diff
@@ -1,158 +1,135 @@
...
+      <ui-marquee
+        {% if section.settings.pause_on_hover %}
+          data-pause-on-hover
+        {% endif %}
+        speed="{{ section.settings.speed }}"
+      >
+        <div ui-marquee="track">
+          <ul ui-marquee="cloud">
+            {% for item in section.blocks %}
+              <li ui-marquee="item">
```
Comment (ibrilBadreddine) [inline themes/kinder/sections/announcement-bar.liquid]:
```suggestion
              <li ui-marquee="item" {{ item.youcan_attributes }}>
```

File: themes/kinder/snippets/_config.liquid [hunk 3]
```diff
@@ -104,13 +104,13 @@
...
 {% endcomment %}
 <script src="https://unpkg.com/@youcan/js@latest/dist/index.umd.js" defer></script>
-{{ 'misc.motion.js' | asset_url | script_tag_deferred }}
 {{ 'misc.toast.js' | asset_url | script_tag_deferred }}
 {{ '_global.js' | asset_url | script_tag_deferred }}
 {{ '_pubsub.js' | asset_url | script_tag_deferred }}
 {{ '_cart.js' | asset_url | script_tag_deferred }}
 {{ 'misc.select.js' | asset_url | script_tag_deferred }}
 {{ 'misc.carousel.js' | asset_url | script_tag_deferred }}
+{{ 'misc.marquee.js' | asset_url | script_tag_deferred }}
```
Comment (ibrilBadreddine) [inline themes/kinder/snippets/_config.liquid:113]: not sure, but if we're using it only with the `announcement bar` section, just move it there

Comment (eihabkhan) [reply themes/kinder/snippets/_config.liquid:113]: it will be used for logo and feature marquees 

SCR-20260427-nhep

File: themes/kinder/sections/announcement-bar.liquid (hunk 2 above)
Comment (eihabkhan) [reply themes/kinder/sections/announcement-bar.liquid]: good catch

**Review by ibrilBadreddine — APPROVED** [review]
